index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <nav>
+      <h3>Freecodecamp Survey Form</h3>
+      <style>
```

Comment:
Please separate the stylesheet into a different folder.

Reply:
- [ ] It's important to add a reason why a change should be made in comments as this lets the other person know why it is being requested.
- For example, you could say "Please separate your stylesheet from your Html file as this makes your codebase cleaner and structured allowing other developers you might collaborate with find it easy to understand your code"